### PR TITLE
docs: add DID query section

### PR DESCRIPTION
**Pull request by ntn-x2**
## fixes KILTProtocol/ticket#2170

### CODE REVIEW

File: docs/develop/01_sdk/02_cookbook/01_dids/05_full_did_query.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+title: Resolve a DID
+---
+
+import TsJsBlock from '@site/src/components/TsJsBlock';
+
+import FullDidQuery from '!!raw-loader!@site/code_examples/core_features/did/06_full_did_query.ts';
+
+Querying the state of a DID is also called **resolution**.
+
+Resolving a DID to its DID Document can be done via the KILT resolver.
```
Comment (weichweich) [inline docs/develop/01_sdk/02_cookbook/01_dids/05_full_did_query.md]: What's a KILT resolver? Is that the SDK?

**Review by weichweich — APPROVED** [review]
LGTM, i just don't know what a resolver is. We might want to explain that.

File: docs/develop/01_sdk/02_cookbook/01_dids/05_full_did_query.md (hunk 1 above)
Comment (ntn-x2) [reply docs/develop/01_sdk/02_cookbook/01_dids/05_full_did_query.md]: Good point. Is this better? Or do you expect a more thorough explanation? https://github.com/KILTprotocol/docs/pull/174/commits/6afa0cf964c68da30bd360e5e9eb5a89be532e59

**Review by weichweich — APPROVED** [review]
thx

**Pull request merged**